typescript/infra/scripts/utils.ts

```diff
@@ -28,7 +28,7 @@ export function assertEnvironment(env: string): DeployEnvironment {
 export function getCoreEnvironmentConfig<Env extends DeployEnvironment>(
   env: Env,
 ) {
-  return environments[env] as any; // TODO: make indexed union compatible
+  return environments[env];
```

Comment:
curious how you'd fell about configuring prettier to remove semicolons?

Reply:
don't feel strongly either way